src/clj_ldap/client.clj

```diff
@@ -152,17 +145,17 @@
   [{:keys [host ssl? start-tls?] :as options}]
   (let [h (host-as-map host)
         opt (connection-options options)
+        ssl-factory (create-ssl-factory options)
```

Comment:
will this create it every time? should you only create this if start-tls or ssl is set to true?

Reply:
I will inline this.